### PR TITLE
fix: Added alt attribute to images in `bootcamp.md` 

### PR DESCRIPTION
**Pull request by developer-diganta**
## 👨‍💻 Changes proposed
Added alt attribute to the bootcamp section images.

## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: pages/Bootcamp.md [hunk 1]
```diff
@@ -5,22 +5,22 @@
...
+|**Day 6** -  (06-04-2022)|<img src="https://user-images.githubusercontent.com/51878265/161997457-b1965b04-d43b-42fd-a1c6-5f4b90b579e5.png" alt="Day 6: Pull Request (PR)" height="300">| [`Twitter`](https://twitter.com/pradumna_saraf/status/1511596794987552771?s=20&t=HSO5HvApxyRyXhVTCN1cvg) | [`Blog`](https://blog.pradumnasaraf.co/what-is-pull-request-pr) |
+|**Day 7** -  (07-04-2022)|<img src="https://user-images.githubusercontent.com/51878265/162221303-45f9cc47-2021-4a91-bb82-1481c7bf75d6.png" alt="Day 7: Issues" height="300">| [`Twitter`](https://twitter.com/pradumna_saraf/status/1511979715137269762?s=20&t=qpehHoByWf9HYwNLgIq5Eg) | [`Blog`](https://blog.pradumnasaraf.co/what-are-the-issues) |
+|**Day 8** -  (08-04-2022)|<img src="https://user-images.githubusercontent.com/51878265/162553119-2d600feb-2943-43d6-be71-6acc771cf709.png" alt="Day 8: Branches"height="300">| [`Twitter`](https://twitter.com/pradumna_saraf/status/1512331039468756996?s=20&t=WLR8fYIFIzzAxUquYfGIKw) | [`Blog`](https://blog.pradumnasaraf.co/what-are-branches) |
```
Comment (Pradumnasaraf) [inline pages/Bootcamp.md:16]:
```suggestion
|**Day 8** -  (08-04-2022)|<img src="https://user-images.githubusercontent.com/51878265/162553119-2d600feb-2943-43d6-be71-6acc771cf709.png" alt="Day 8: Branches" height="300">| [`Twitter`](https://twitter.com/pradumna_saraf/status/1512331039468756996?s=20&t=WLR8fYIFIzzAxUquYfGIKw) | [`Blog`](https://blog.pradumnasaraf.co/what-are-branches) |
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,
Thank you for the PR. There are some changes required to merge this PR successfully.

That causes the image not well-aligned issue
Screenshot from 2022-09-18 16-50-22

**Comment by developer-diganta** [comment]
@Pradumnasaraf I have done the suggested changes. Can you please check if it works correctly?

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**